webpack.config.ts

```diff
@@ -85,12 +85,12 @@ export default {
           },
         ],
       },
-      {
-        test: /\.m?js/,
-        resolve: {
-          fullySpecified: false,
-        },
-      },
+      // {
+      //   test: /\.m?js/,
+      //   resolve: {
+      //     // fullySpecified: false,
+      //   },
+      // },
     ],
```

Comment:
Before commenting out this was causing crashes, on a clean environment setup. @damiankoper  please take a look.

Reply:
The purpose of this was to keep benchmark.js sane. With this lib gone, there is no need to keep this setting. Can be removed.

Reply:
Removed this piece of code.